packages/@aws-cdk/aws-lambda-python-alpha/test/bundling.test.ts

```diff
@@ -322,13 +322,27 @@ test('Bundling a function with poetry dependencies, with hashes', () => {
       ],
     }),
   }));
+});
 
-  const files = fs.readdirSync(assetCode.path);
-  expect(files).toContain('index.py');
-  expect(files).toContain('pyproject.toml');
-  expect(files).toContain('poetry.lock');
-  // Contains hidden files.
-  expect(files).toContain('.ignorefile');
```

Comment:
Any reason for removing these assertions from the above test case?

Reply:
I thought there was a good reason for this but I can't remember why—I added it back.